Tasks/Common/azure-arm-rest-v2/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "azure-arm-rest-v2",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
I don;t think we do update the common package version

Reply:
+1